L1Trigger/L1THGCal/test/HGCalBackendStage1ParameterExtractor.cc

```diff
@@ -275,6 +280,14 @@ uint32_t HGCalBackendStage1ParameterExtractor::getTCaddress(int& tc_ueta, int& t
   }
 }
 
+uint32_t HGCalBackendStage1ParameterExtractor::getReducedModuleHash(int subdet, int layer, int ueta, int vphi) {
```

Comment:
Might be better to take directly the detid as input parameter. This way one could use `HGCalTriggerTools` function like `isScintillator()` and `layerWithOffset()`